corehq/apps/app_manager/models.py

```diff
@@ -5728,64 +5730,23 @@ class SQLGlobalAppConfig(models.Model):
     class Meta(object):
         unique_together = ('domain', 'app_id')
 
-
-class GlobalAppConfig(Document):
-    # this should be the unique id of the app (not of a versioned copy)
-    app_id = StringProperty()
-    domain = StringProperty()
-
-    # these let mobile prompt updates for application and APK
-    app_prompt = StringProperty(
-        choices=["off", "on", "forced"],
-        default="off"
-    )
-    apk_prompt = StringProperty(
-        choices=["off", "on", "forced"],
-        default="off"
-    )
-
-    # corresponding versions to which user should be prompted to update to
-    apk_version = StringProperty(default=LATEST_APK_VALUE)  # e.g. '2.38.0/latest'
-    app_version = IntegerProperty(default=LATEST_APP_VALUE)
-
     @classmethod
     def for_app(cls, app):
-        """
-        Returns the actual config object for the app or an unsaved
```

Comment:
Noting that the new implementation changes this, so that calling `for_app` will create and save a new object if none exists. Given that these docs are small and there's only one per app, I think this is fine, but fyi @sravfeyn